docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:collection                | String Literal     | N/A   | A String Literal that represents the physical archival collection the resource belongs to. |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:isPartOf                  | URI                | N/A   | A URI that represents the physical archival collection a resource belongs to.              |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:memberOfArchivalSeries | URI/String Literal | N/A   | A URI or String Literal that indicates the archival series a resource belongs to.          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:sheetMusic_hostItem    | URI/String Literal | N/A   | A URI or String Literal that indicates the host item for the resource being described.     |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| relators:[term]               | URI/String Literal | N/A   | Use with a role from MARC Code List of Relators role terms.                                |
+|                               |                    |       | Value is either text or URI from a controlled vocabulary (like                             |
+|                               |                    |       | Library of Congress Name Authority File).                                                  |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+
+relatedItem as simple structural metadata
+-----------------------------------------
+Use Case
+^^^^^^^^
+`relatedItem`, with and without attributes, is used in a number of collections to express structural relationships. These relationships will be handled/expressed by default behavior in our next-gen DAMS. In the case of `relatedItem/abstract`, the values present should be handled at the collection level.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[not(@*)]` OR
+:code:`relatedItem[@type='host'][@displayLabel[matches(., 'project') or matches(., 'Project') or matches(., 'Digital Collection') or matches(., 'Project Part') or matches(., 'Is Part Of')]` OR
+:code:`relatedItem/abstract`
+
+Decision
+^^^^^^^^
+Do not migrate.
+
+relatedItem[@type='host'][@displayLabel='Collection']
+-----------------------------------------------------
+Use Case
+^^^^^^^^
+This XPath is used to indicate the resource's archival collection.
```

Comment:
If you want to expand upon the use case, you could include some more information about how these values are (or potentially might be shared). We do share this as a string in DPLA, though currently nothing is done with it. I know Gretchen had worked on a white paper on including collection names in DPLA that doesn't seem to have come to fruition on the actual platform - https://docs.google.com/document/d/16r_px4GajLIOZMlyXyN0pJFfWpstJIKxHF3brgM4YOo/editI think I might've mentioned this in a comment on your initial mapping doc.

In terms of actual current use, we actually include archival collection as a facet using utk_mods_archColl_archID_ms. It does also display in the Click for details section.